flask_app/src/services/base.py

```diff
@@ -31,13 +31,13 @@ def get_all_entities(entity: db.Model, **filters):
     try:
         entity_list = entity.get_all_by(**filters)
         if not entity_list:
-            return {'message': "No entities found or it's empty"}, 404
+            return {'message': "No entities found"}, 204
         return {f"all_{entity.__tablename__}": entity_list}, 200
     except Exception as e:
         return {'error': str(e)}, 500
 
 
-def search_entities(entity, search: str, *columns):
+def search_entities(entity, search: int, columns):
```

Comment:
Va falloir que je vous explique aussi celle-ci, et surtout que j'écris les unit tests

Reply:
Ah yes erreur de ma part sur celle-ci désolé, c'était pour tester quelque chose
```suggestion
def search_entities(entity, search: str, *columns):
```

Reply:
A fond d'accord sur les tests, je me base sur les existants pour voir s'il y a du cassant. 
Hésite pas à faire rapidement ton commit pour éviter qu'on ai trop de diffs ;)